velox/vector/tests/VectorTestBase.h

```diff
@@ -509,4 +508,34 @@ class VectorTestBase {
   velox::test::VectorMaker vectorMaker_{pool_.get()};
 };
 
+/// Wraps input in a dictionary that reverses the order of rows.
+class TestingDictionaryFunction : public exec::VectorFunction {
```

Comment:
Thank you for this suggestion! I moved the `TestingDictionaryFunction` class into velox/vector/tests/TestDictionaryUtils.h. I'll refactor other uses of it in a separate PR since they are not closely related to the current one.

Reply:
Thank you. It would be nice to name the file after the class, e.g. TestingDictionaryFunction.h/cpp.